src/components/MyOfferDetails.vue

```diff
@@ -0,0 +1,449 @@
+<template>
+  <v-toolbar color="blue" extended extension-height="100">
+    <v-chip @click="goByLink(`/module/${this.role}/tenders`)"
+    style="margin-left: 12rem" variant="text" text-color="white" prepend-icon="mdi-keyboard-backspace">Back
+    </v-chip>
+    <template v-slot:extension>
+      <v-container class="px-15">
+        <v-toolbar-title class="ml-14 mb-4" style="font-size: 1.5rem">{{ offer.organizationNameByBidder }}</v-toolbar-title>
+        <v-tabs v-model="tab" height="30" class="mb-10 ml-12" :color="tabColor">
+          <v-tab v-if="offer.bidderSt === 'Offer selected by Contractor' || offer.bidderSt === 'Contract declined by Bidder' || offer.bidderSt === 'Contract approved by Bidder'"
+          value="awardDecision">Award Decision</v-tab>
+          <v-tab v-if="offer.bidderSt === 'Offer rejected by Contractor'" value="rejectDecision">Reject Decision</v-tab>
+          <v-tab value="tenderDescription">Tender Description</v-tab>
+          <v-tab value="myOffer">My Offer</v-tab>
+        </v-tabs>
+      </v-container>
+    </template>
+  </v-toolbar>
+
+  <v-window v-model="tab" class="mt-n7 pb-10">
+    <v-window-item value="awardDecision">
+      <v-card class="mx-auto" elevation="8" max-width="1000">
+        <v-toolbar color="white" height="280" class="text-left">
+          <v-toolbar-title v-if="offer.bidderSt === 'Offer selected by Contractor'" class="text-center" style="font-size: 1.5rem">
```

Comment:
the following v-ifs are better to replace with the v-toolbar-title that would get value from a computed property. put all if else logic into that computed property